source/_integrations/uptimerobot.markdown

```diff
@@ -25,4 +25,8 @@ The **UptimeRobot** {% term integration %} provides entities to get the status f
 
 To get your API key, go to [My Settings](https://uptimerobot.com/dashboard#mySettings) on the UptimeRobot website, at the bottom you will find your "Main API Key".
 
+> **NOTE**  
+> If you get redirected to the [new dashboard](https://dashboard.uptimerobot.com), you won't be able to find the API key.  
+> Go to [Account Details](https://dashboard.uptimerobot.com/account/details) and **disable** `Use New Interface as Default`. You'll now be able to use the previous link to My Settings and create the key.
+
```

Comment:
```suggestion
```

Using the specific URL to get the old dashboard is enough (like suggested [here](https://github.com/home-assistant/home-assistant.io/pull/32842/files#r1606457687)).
For now at least... Looking at how they have their new dashboard, it might be nearing the end of this integration..

Reply:
Oh I didn't see that domain, nice. I'll revert all this then, thanks.
Yes, even their documentation still refers to My Settings which doesn't exist in new dashboard.
I'll drop them an email to see what's the plan (missing feature or deprecated)

Reply:
Just pinged chat support and they said it's a known missing feature, not meant to be deprecated or moved to paid-only. It will be available in the new UI, this integration should survive 😃